code/game/machinery/computer/abnormality_work.dm

```diff
@@ -214,6 +214,43 @@
 	// Initial chance and speed values before work started, in case they get overriden by abnormality
 	var/init_work_chance = work_chance
 	var/init_work_speed = work_speed
+	var/agent_stat_total = 0
+	var/work_agent_stat_total = 0
+	var/agent_count = 0
+	for(var/mob/living/carbon/human/H in AllLivingAgents())
+		if(!H.client)
+			continue
+		if(!H.mind)
+			continue
+		if(!H.z = z) //To prevent things like thunderdome from interfering from stat gain.
```

Comment:
Ya gotta do == or else it causes intigration failures. I dont know why this isnt showing up as a linter issue.

Reply:
...How did I manage to make that stupid mistake....